chat-plugins/trivia.js

```diff
@@ -408,7 +389,14 @@ var Trivia = (function () {
 
 	return {
```

Comment:
``getInstance``'s job is to avoid having to create the Trivia object until it is first needed, then return the cached instance afterwards when its methods are needed instead of having to call the instance's constructor every time, or to immediately create an instance at runtime and return that every time.